src/software/redis-server/redis_test.cpp

```diff
@@ -0,0 +1,38 @@
+#include <iostream>
+#include <string>
+
+#include "cpp_redis/core/client.hpp"
+#include "software/test_util/test_util.h"
+
+TEST(ServerTest, Get)
```

Comment:
nit: maybe rename to something like `(RedisKeyValueStoreTests, cpp_redis_get_test)`